cover_agent/cover_agent.py

```diff
@@ -316,7 +316,7 @@ def finalize_test_generation(self, iteration_count):
         if self.test_validator.current_coverage >= (desired_coverage / 100):
             self.logger.info(
                 f"Reached above target coverage of {desired_coverage}% "
-                f"(Current Coverage: {current_coverage}%) in {iteration_count} iterations."
+                f"(Current Coverage: {current_coverage}%) in {iteration_count + 1} iterations."
```

Comment:
This fixes a situation when a desired coverage is reached during the first iteration and it looks like it is reached in 0 iterations.